src/aiida_quantumespresso/parsers/pw.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 """`Parser` implementation for the `PwCalculation` calculation job class."""
+import os
```

Comment:
Not used I believe
```suggestion
```

Reply:
Yes, I imported it cause at the beginning I put ``CRASH`` in the temporary folder. Don't know why the linter did not detect it.